packages/amplication-git-utils/src/types.ts

```diff
@@ -33,6 +33,26 @@ export interface GitProvidersConfiguration {
   bitBucketConfiguration: BitBucketConfiguration;
 }
 
+interface OAuthProviderOrganizationProperties extends OAuthData, CurrentUser {
+  links: { avatar: LinksMetadata };
+  username: string;
+  useGroupingForRepositories: boolean;
+  uuid: string;
+  displayName: string;
+  accessToken: string;
+  refreshToken: string;
+  tokenType: string;
+  expiresAt: number;
+  scopes: string[];
+}
+
+export type BitBucketProviderOrganizationProperties =
+  OAuthProviderOrganizationProperties;
+
+export interface GitHubProviderOrganizationProperties {
+  installationId: string;
+}
+
 export interface GitProviderArgs {
   provider: EnumGitProvider;
   providerOrganizationProperties: any;
```

Comment:
Shouldn't we connect both (/all) types of *OrganizationProperties so that we don't use _any_ as the type for _providerOrganizationProperties_ ?

Reply:
I agree